content/docs/user-guide/programming/gems/_index.md

```diff
@@ -1,5 +1,7 @@
 ---
-title: Gem Development
+linkTitle: Gem Development
+title: Programmer's Guide to Gem Development
+description: Learn how to create your own Gems in Open 3D Engine (O3DE).
 weight: 200
```

Comment:
Let's make sure gem and component development appear near the top of the ToC, and in that order (to match their order of appearance in the gem and component reference sections).
```suggestion
weight: 120
```